backend/app.js

```diff
@@ -21,6 +21,32 @@ app.get('/exercises', async (req, res)=>{
         console.error(error.message);
     }
 })
+//2) create exercise
```

Comment:
didn't work well, currently on freeze, will continue working with mockDB until find a way to correctly store code